tests/unit/test_parameters/test_process_parameter_data.py

```diff
@@ -2,56 +2,59 @@
 # Tests for the parameter processing functions
 #
 
-
 import os
 import numpy as np
 import pybamm
-
 import pytest
 
 
+@pytest.fixture
+def path():
+    return os.path.abspath(os.path.dirname(__file__))
```

Comment:
This can be made more readable – it can be simplified to

```suggestion
def path():
    return Path(__file__).parent.resolve()
```

for which you'll also need a `from pathlib import Path` statement at the top of the file.